python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/test_container_context.py

```diff
@@ -89,6 +89,14 @@ def test_merge(
         "ephemeral_storage": 25,
     }
 
+    assert merged.server_ecs_tags == [
+        {"key": "BAZ", "value": "QUUX"},
+        {
+            "key": "FOO",  # no value
+        },
+    ]
+    assert merged.run_ecs_tags == [{"key": "GHI"}, {"key": "ABC", "value": "DEF"}]
```

Comment:
test that verifies when you merge two container contexts together they add to each others lists